src/includes/java/performance/getting-started/java.mdx

```diff
@@ -4,6 +4,17 @@ Performance Monitoring is available for the Sentry Java SDK version ≥ 4.0.
 
 Sentry allows you to monitor the performance of your application, showing you how latency in one service may affect another service, and letting you pinpoint exactly which parts of a given operation may be responsible. To do this, it captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services, respectively. You can learn more about this model in our [Distributed Tracing](/product/performance/distributed-tracing/) docs.
 
+<Note>
```

Comment:
this should be under either a `includes` or `supported` feature flag, we don't want this visible for Android docs.
https://github.com/getsentry/sentry-docs/pull/2970 probably solves this by separating things

Reply:
So lets wait till #2970 is merged.